pages/common/xsv.md

```diff
@@ -0,0 +1,27 @@
+# xsv
+
+> A fast CSV command line toolkit written in Rust.
+
+- Inspect the headers of a file:
+
+`xsv headers {{path/to/file.csv}}`
+
+- Count the number of entries:
+
+`xsv count {{path/to/file.csv}}`
+
+- Get an overview of the shape of entries:
+
+`xsv stats {{path/to/file.csv}} | xsv table`
+
+- Select a few columns:
+
+`xsv select {{column_a,column_b}} {{path/to/file.csv}}`
+
+- Show 10 random entries:
+
+`xsv sample {{10}} {{path/to/file.csv}}`
+
+- Join a column from one file to another:
+
+`xsv join --no-case {{key_a}} {{path/to/file/a.csv}} {{column_b}} {{path/to/file/b.csv}} | xsv table`
```

Comment:
Question: Why is the first one called key_ and the second one column_ ? Shouldn't both be "column_x"

Reply:
I was trying to hint at what join does but maybe its more confusing than not? will switch

Reply:
If it is like a sql-join where values of one column should be equal to values of another column, then yes I think column_a and column_b sounds better. I was curious to know why you mentioned "key".